app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import NotVaccinatedError, OutdatedVaccineError,\
+    NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError
+        if visitor["wearing_a_mask"] is False:
+            raise NotWearingMaskError
```

Comment:
The `NotWearingMaskError` should include a descriptive message, such as 'Visitor is not wearing a mask.'